pages/learn/develop/blocks.md

```diff
@@ -0,0 +1,119 @@
+
+For example, getting a browser to run well in a container is a tedious task. So, why not use the [browser block][browser].
+
+
+<img src="/img/blocks/diagram-1.png" width="100%">
+
+## Develop with a block
+
+Blocks being a container image can be added to a multi-container app. To use a block in your app, add the block's image as a new service in the `services` section of your app's `docker-compose.yml` file. This is similar to how we would [build multicontainer apps][multicontainer].
+
+Here's an example of using the [browser block][browser] in a sample multi-container app running 2 services. More blocks 
+
+```
+version: '2'
+
+services:
+  # Browser block service goes here 
+  browser:
+    image: balenablocks/browser
+    privileged: true 
+    network_mode: host
+
+  # Existing services in the multi-container app
+  balena-node-hello-world:
+    build: ./balena-node-hello-world
+    restart: always
+    depends_on:
+      - browser
+    ports: 
+      - "80"
+  importantBananaDb:
+    image: importantBanana/db:latest
+    ports: 
+      - "8080"
+    environment:
+      DB_ROOT_PASSWORD: 'potassium'
+      RUN_MODE: 'super-secure'
+``` 
+
+Similarly, each block will have be having their own set of [docker-compose fields][docker-compose] that it requires for configuration and usage. Do take care of syntax and conflicts in the docker-compose.yml file when composing and deploying your multi-container configuration. 
+
+
+More resources: 
+
+
+1. Video tutorial on [IoT Happy Hour](https://youtu.be/Mllay6Z2-qQ?t=1440) about using  balenaBlocks and building an [environmental sensor app](https://github.com/balenalabs-incubator/ruuvitag) using sensor, connector, and dashboard block.
+   
+2. Check out our guide on [how to compose multi-container apps](https://www.balena.io/blog/two-projects-one-device-turn-your-raspberry-pi-into-a-multitool/) for more details. 
+   
+3. Blog on [Introduction to blocks](https://www.balena.io/blog/introducing-balenablocks-jumpstart-your-iot-app-development/).
+
+## Creating your own block 
+
+Since blocks are container images, the process to create blocks is very similar to creating a container image. To learn more about the what can be a block, you can read this [blog post](https://www.balena.io/blog/how-to-make-your-own-balenablocks-simple-drop-in-edge-app-functionality/). In order to create your own block:
+
+1. The code for your block should be in GitHub project and has to [include a balena.yml file](https://www.balena.io/docs/learn/deploy/deploy-with-balena-button/#balenayml-configuration-file) and a logo. Here's a balena.yml file of a block for reference:
+
+
+```yml
```

Comment:
Is there a reason to show the capture block here, rather than stick to the browser block?

Reply:
No reason, I got this from the blog, I will swap this out.